apps/component-date/README.md

```diff
@@ -0,0 +1,3 @@
+Date component example
+============
+
```

Comment:
The intention (or at least *my* intention) for these examples is that they would provide tutorial-style content for how a new hof user would go about implementing the included functionality.

These would then be dynamically loaded into the `hof-guide`.

Can you write something like that for this example?